assignments/array-methods.js

```diff
@@ -58,26 +58,62 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 let fullName = [];
 console.log(fullName);
 
+runners.forEach((getNames)=>{
+    let names = getNames.first_name + " " + getNames.last_name;
+    fullName.push(names)
+  });
+
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
 console.log(allCaps); 
 
+caps.push(capitalize.first_name.toUpperCase())
+const caps = runners.map((capitalize));{
```

Comment:
You have a syntax error with the semi-colon. The arrow syntax is:

```js
runners.map(() => {
    /* Do stuff */
});
```

You may also use the function keyword

```js
runners.map(function() {
    /* Do stuff */
})
```

Please reach out to me if you have any questions about this. I sense some struggle here, and I'd like to help ease that as much as possible.